src/_pytest/config/__init__.py

```diff
@@ -1662,6 +1663,54 @@ def _warn_about_skipped_plugins(self) -> None:
             )
 
 
+class OutputVerbosity:
+    DEFAULT = "auto"
+    _option_name_fmt = "verbosity_{}"
+
+    def __init__(self, config: Config) -> None:
+        self._config = config
+
+    @property
+    def verbose(self) -> int:
```

Comment:
Like the class name, I'm open tweaking the name. I went with `verbose` to make the command line option name destination (which is how most things access the application wide value now). But `pytest` is already inconsistent:

* command line options name: `--verbosity`
* command line options destination name: `verbose`
* `TerminalReporter` property: `verbosity`

Maybe a follow up cleanup PR would be to make the code base more consistent.

Reply:
I suggest we just remove this option, and let users access it via `.get()`.

Note: I removed and pushed this property in a separate commit, we can revert it if we decide it otherwise.